scripts/lib/mkcloud-onhost.sh

```diff
@@ -121,7 +121,7 @@ function onhost_cacheclouddata
         for a in $architectures; do
             local suffix
             # Mirror SLES/HA/SOC update + pool repos
-            for suffix in Pool Updates; do
+            for suffix in Pool Updates Updates-test; do
```

Comment:
@matelakat @dirkmueller the above means $hacloud have a dependency on $want_test_updates. In other words, both options must be enabled. Is that something we want or can they be separated?